src/scripts/clean.sh

```diff
@@ -95,7 +95,13 @@ destroy_mlz() {
 
   # clean up files
   delete_files_in_directory_by_name "${src_path}" "${tfvars_file_name}"
-  rm -rf "${configuration_output_path}/${mlz_env_name}.mlzconfig" "${configuration_output_path:?}/${tfvars_file_name}"
+  echo "INFO: deleting ${configuration_output_path}/${mlz_env_name}.mlzconfig ..."
+  rm -rfv "${configuration_output_path}/${mlz_env_name}.mlzconfig" "${configuration_output_path:?}/${tfvars_file_name}"
```

Comment:
@glennmusa 
I noticed this. Is this required or can we just leave it as:
rm -rfv "${configuration_output_path}/${mlz_env_name}.mlzconfig"

I think the previous line that executes the delete_files_in_directory_by_name function takes care of all tfvars files?

Reply:
You're right, we can allow `delete_files_in_directory_by_name` passing the `src_path` and the `tfvars_file_name` to delete the tfvars file. Good catch.

Don't think we need the verbose `-v` option here neither.

Want to go ahead and make those changes? Otherwise this looks good to me. Thanks!